content/blog/2017/2017-04-10-welcome-to-blue-ocean-pipeline-activity.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean's Pipeline Activity View"
+tags:
+- blueocean
+- ux
+- pipeline
+- tutorial
+- screencast
+
+author: lnewmanå
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:https://cloudbees.com[CloudBees].
+
+Blue Ocean is a new user experience for Jenkins,
+and version 1.0 is now live!
+Blue Ocean makes Jenkins, and continuous delivery, approachable to all team members.
+In my link:/blog/2017/04/06/welcome-to-blue-ocean-editor[previous post],
+I showed how easy it is to create and edit Declarative Pipelines using the Blue Ocean Pipeline Visual Editor.
+In this video, I'll use the Blue Ocean Pipeline Activity View to track the
+state of branches and Pull Requests in one project.
+Blue Ocean makes finding the logs I need to triage Pipeline failures so much easier.
```

Comment:
s/Pipeline failures/failures/

Something about this phrasing makes it seem like Pipeline fails often, rather than failures in "my" stuff